browser/src/unocommands.js

```diff
@@ -276,6 +276,7 @@ var unoCommandsArray = {
 	'InsertFootnote':{text:{context:_('Insert Footnote'),menu:_('~Footnote'),},},
 	'InsertFrame':{text:{context:_('Insert Frame'),menu:_('F~rame...'),},},
 	'InsertGraphic':{global:{context:_('Insert Image...'),menu:_('~Image...'),},},
+	'InsertAVMedia':{global:{context:_('Insert Multimedia...'),menu:_('Multimedia...'),},},
```

Comment:
@Minion3665:
Manual update of this file is prohibited. In core co-24.04 branch the strings are different, as it turned out when I updated UNO command translations.
```
'InsertAVMedia':{global:{context:_('Insert Audio or Video'),menu:_('Media'),},},
```
But then, the "Local Multimedia" menu item will differ. This has to be sorted out in a follow-up patch.